kornia/models/__init__.py

```diff
@@ -1,2 +1,8 @@
-from . import detection, segmentation, tracking
+from . import (
+    depth_estimation,
```

Comment:
Should we have the most basic which is image classification? And eg. Just embedding extraction, and support let’s say Dinov2?

Reply:
And wondering how can we fit here in terms of api multimodal or foundational models with multi task heads

Reply:
Super resolution is another one https://pytorch.org/tutorials/advanced/super_resolution_with_onnxruntime.html

Maybe add as todos